pkgs/development/libraries/rnnoise/default.nix

```diff
@@ -1,16 +1,27 @@
-{ stdenv, lib, fetchFromGitHub, autoreconfHook }:
+{ stdenv, lib, fetchFromGitHub, fetchurl, autoreconfHook }:
 
 stdenv.mkDerivation (rec {
   pname = "rnnoise";
-  version = "2021-01-22";
+  version = "0.2";
 
   src = fetchFromGitHub {
```

Comment:
Supporting both seemed extravagant. While I understand "Microsoft GitHub" is proprietary instance, I don't think it matters in case of nixpkgs. In fact [many packages](https://github.com/search?q=repo%3ANixOS%2Fnixpkgs%20fetchFromGitHub&type=code) in nixpkgs utilise GitHub to host their source code.

I am more inclined to use a "universal" git instance (that can be either official GitLab or GitHub). This reduces the chances of downtimes than using a "personal" mirror of the organisation.

Reply:
Good point. I'll switch to `fetchFromGitLab` instead.

Reply:
Valid point.

Instead of using `fetchurl.urls` to specify two urls of zips, is there room to use `fetchFromGitLab` and `fetchFromGitHub` in conjuction, somehow?

Reply:
Done.